### ISSUE TITLE
Catch errors in deploy.yml ERB better

### ISSUE THREAD

**Issue by dhh** [mrsked/mrsk#6]
If you make a mistake in deploy.yml, especially with the ERB definitions, like referencing missing credentials, the error message is obtuse and the backtrace hard to parse. Let's improve this and make it easier to know exactly what went wrong.

**Comment by kjellberg** [comment]
What about validating `raw_config` with json-schema?

https://github.com/voxpupuli/json-schema

**Comment by dhh** [comment]
Interesting idea. Worried it might be too heavy, but we could try!

**Comment by dhh** [comment]
Don't actually need ERB now by default. So not really relevant. Still should validate the schema, though.
